CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [7.5.1] - 2024-10-22
+### Added
+- Added `apiary_extra_tags_s3` variable to add s3 buckets extra tags.
```

Comment:
did you update the README to add the new variable?

Reply:
Done, thanks for the heads up !